### ISSUE TITLE
Cannot handle multiple paths as args

### ISSUE THREAD

**Issue by jvacek** [greut/eclint#10]
Hey there!
I wanted to create an issue here just in case you're not active on GitLab anymore, but I'm trying to create a pre-commit hook so i can use `eclint` in my repos. ([see here](https://gitlab.com/greut/eclint/-/merge_requests/155#note_2133192867))

`eclint` will happily build and run in the pre-commit environment, and there's not issue in the "distribution" process, however I am not able to get an optimal result due to an underlying issue in `eclint` that I can reproduce outside of the environment.

**tldr**, `eclint -fix path1 path2` will only fix `path1`. I see the same behaviour without the `-fix` flag.

It would be nice if `eclint` could handle _all_ paths given to it via args.

**Comment by greut** [comment]
Thanks @jvacek; I'll take a look.
